ts/components/messages/paginated/MessagesSearch.tsx

```diff
@@ -29,12 +29,17 @@ const MessagesSearch = ({
   renderSearchResults,
   searchText
 }: Props) => {
-  const searchResults: ReadonlyArray<UIMessage> = messages.filter(message =>
-    isTextIncludedCaseInsensitive(
-      [message.title, message.organizationName, message.serviceName].join(" "),
-      searchText
+  const searchResults: ReadonlyArray<UIMessage> = messages
+    .filter(message =>
+      isTextIncludedCaseInsensitive(
+        [message.title, message.organizationName, message.serviceName].join(
+          " "
+        ),
+        searchText
+      )
     )
-  );
+    // sorted from newest to oldest
+    .sort((a, b) => b.createdAt.getTime() - a.createdAt.getTime());
```

Comment:
⛏️ **nitpick**: we (as in "the backend") sort the messages by ID, which happens to be monotonic and match the sort by time. I would rather use ID though to avoid confusion.
You can find an example [here](https://github.com/pagopa/io-app/blob/f8487a50d5e27dbfd1810481e46dbe5cd3b37677/ts/store/reducers/entities/messages/index.ts#L56). Mind that [localeCompare](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare) should probably be forced to `en` to match the backend logic. Roping in @balanza to kindly confirm this.

Reply:
Yes, the ID order matches the creation time order, so you can compare IDs as it might be handy.

Reply:
Done with 6d121aad.